demo/inspect_electrical_series/CMakeLists.txt

```
@@ -0,0 +1,63 @@
+cmake_minimum_required(VERSION 3.15)
+project(nwb_analysis_demo VERSION 0.1.0 LANGUAGES CXX)
+
+# Set C++ standard
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+# Allow configuring paths to dependencies
+set(AQNWB_DIR "${CMAKE_CURRENT_SOURCE_DIR}/../../build/dev" CACHE PATH "Path to aqnwb build directory")
+set(HDF5_DIR "${CMAKE_CURRENT_SOURCE_DIR}/../../libs/hdf5_build/install/cmake" CACHE PATH "Path to HDF5 build directory")
```

```suggestion
set(HDF5_ROOT "${CMAKE_CURRENT_SOURCE_DIR}/../../libs/hdf5_build/install/cmake" CACHE PATH "Path to HDF5 build directory")
```

I think HDF5_ROOT is the environment variable used by CMake in FindHDF5. This is also what we tell users to set in the main aqnwb installation instructions

---

Done

---

I reverted this back to HDF5_DIR because otherwise the GitHub action was failing. I think the differnce is that `HDF5_DIR` points to the cmake folder and `HDF5_ROOT` points to the main HDF5 folder such that cmake searches for the build. By setting HDF5_ROOT in the CMakeLists.txt it confuses the workflow, but by using HDF5_DIR here we can set it to a custom path and cmake can still find HDF5 based on HDF5_ROOT if HDF5 is in a different location. The alternative would be remove the set calls foir HDF5_DIR and BOOST_DIR here  and always require the user to set them.

---

I see. I don't think it's super important to change in this PR since this is just a demo, but I would lean towards having the user set them if needed (especially since a specific version of boost is currently specified as the default). In the workflows, the BOOST_ROOT set in the CMakeLists.txt file is currently being ignored for compatibility reasons and I think if we remove the default path for HDF5_DIR in the CmakeLists.txt file, then Cmake should be able to find HDF5 (as in the test workflows)?